### PR TITLE
Adding tasks about Arrays

### PR DESCRIPTION
**Pull request by TamaraVanyan**

### CODE REVIEW

File: TasksAboutArrays/ArrayIndexToValue.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        int[] array = new int[n];
+        if (a < n){
+            array[a] = a;
+        }
+        if (a + 1 < n){
+            array[a + 1] = a + 1;
+        }
+        if (a + 2 < n){
+            array[a + 2] = a + 2;
+        }
```
Comment (David91) [inline TasksAboutArrays/ArrayIndexToValue.java]: Why are these checks, if this point can be reached after the initial validation above?

File: TasksAboutArrays/ArrayInputMinMaxNumbers.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
+package TasksAboutArrays;
+import java.util.Scanner;
+public class ArrayInputMinMaxNumbers {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        int n = scanner.nextInt();
+
+        if(n < 1 || n >8) {
```
Comment (David91) [inline TasksAboutArrays/ArrayInputMinMaxNumbers.java]: Will the code below work in case of n == 2?

File: TasksAboutArrays/ArraySizeWithN.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
+package TasksAboutArrays;
+import java.util.Scanner;
+public class ArraySizeWithN {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        int n = scanner.nextInt();
+
+        if(n == 0 || n < 0){
```
Comment (David91) [inline TasksAboutArrays/ArraySizeWithN.java]: Simplify the condition.

File: TasksAboutArrays/ArraySizeWithN.java [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+        switch (n) {
+            case 3:
+                array = new int[3];
+                break;
+            case 4:
+                array = new int[4];
+                break;
+            case 5:
+                array = new int[5];
+                break;
```
Comment (David91) [inline TasksAboutArrays/ArraySizeWithN.java]: This can be shortened, if you think a bit longer)

**Review by David91 — CHANGES_REQUESTED** [review]

File: TasksAboutArrays/ArrayIndexToValue.java (hunk 1 above)
Comment (TamaraVanyan) [reply TasksAboutArrays/ArrayIndexToValue.java]: I just removed that ifs

File: TasksAboutArrays/ArraySizeWithN.java (hunk 3 above)
Comment (TamaraVanyan) [reply TasksAboutArrays/ArraySizeWithN.java]: I changed it into: if(n <= 0)

File: TasksAboutArrays/ArraySizeWithN.java (hunk 4 above)
Comment (TamaraVanyan) [reply TasksAboutArrays/ArraySizeWithN.java]: just switch (n) {
            case 3:
            case 4:
            case 5:
                array = new int[n];
                break;

**Pull request merged**